items/item.go

```diff
@@ -92,6 +104,11 @@ func (s *Store) Item(id string) (*Item, error) {
 	if result != nil && len(result.Versions) > 0 {
 		return result, nil // a complete item record
 	}
+	// if item not in cache, and useStore disabled, return ErrNoStore Error error
+	if result == nil && s.useStore == false {
```

Comment:
good point. that code is a holdover from when i thought items might be in an "incomplete" cached state. but in fact, if an item is cached, it will have all its versions.